### PR TITLE
add tool-dependencies/unused_paths

### PR DESCRIPTION
**Pull request by bernt-matthias**
i.e. 

```
GET /api/dependency_resolvers/unused_paths
PUT /api/dependency_resolvers/unused_paths
```

lets see if I got this right

### CODE REVIEW

File: bioblend/_tests/TestGalaxyToolDependencies.py [hunk 1]
```diff
@@ -28,3 +28,11 @@ def test_summarize_toolbox(self):
         assert isinstance(toolbox_summary_select_tool_ids, list)
         assert len(toolbox_summary_select_tool_ids) == 1
         assert toolbox_summary_select_tool_ids[0]["tool_ids"][0] == tool_id
+
+    def test_unused_dependency_paths(self):
```
Comment (nsoranzo) [inline bioblend/_tests/TestGalaxyToolDependencies.py:32]:
```suggestion
    @test_util.skip_unless_galaxy("release_20.01")
    def test_unused_dependency_paths(self):
```

File: bioblend/_tests/TestGalaxyToolDependencies.py [hunk 2]
```diff
@@ -28,3 +28,11 @@ def test_summarize_toolbox(self):
...
+        unused_paths = self.gi.tool_dependencies.unused_dependency_paths()
+        assert isinstance(unused_paths, list)
+        assert len(unused_paths) == 0
+
+    def test_delete_unused_dependency_paths(self):
```
Comment (nsoranzo) [inline bioblend/_tests/TestGalaxyToolDependencies.py:37]:
```suggestion
    @test_util.skip_unless_galaxy("release_20.01")
    def test_delete_unused_dependency_paths(self):
```

File: bioblend/_tests/TestGalaxyToolDependencies.py [hunk 3]
```diff
@@ -28,3 +28,11 @@ def test_summarize_toolbox(self):
         assert isinstance(toolbox_summary_select_tool_ids, list)
         assert len(toolbox_summary_select_tool_ids) == 1
         assert toolbox_summary_select_tool_ids[0]["tool_ids"][0] == tool_id
+
+    def test_unused_dependency_paths(self):
+        unused_paths = self.gi.tool_dependencies.unused_dependency_paths()
+        assert isinstance(unused_paths, list)
+        assert len(unused_paths) == 0
```
Comment (nsoranzo) [inline bioblend/_tests/TestGalaxyToolDependencies.py:35]: I'd leave this out in case the tests are run against a real Galaxy server.

```suggestion
```

**Review by nsoranzo — APPROVED** [review]

**Comment by bernt-matthias** [comment]
Tested this on my dev instance.

**Comment by nsoranzo** [comment]
Thanks @bernt-matthias !

**Pull request merged**